xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -178,6 +179,21 @@ def safe_exec(
         else:
             emsg = None
 
+        # Run the code in both the remote codejail service as well as the local codejail
+        # when in darklaunch mode.
+        if is_codejail_in_darklaunch():
+            data = {
+                "code": code_prolog + LAZY_IMPORTS + code,
+                "globals_dict": globals_dict,
+                "python_path": python_path,
+                "limit_overrides_context": limit_overrides_context,
+                "slug": slug,
+                "unsafely": unsafely,
+                "extra_files": extra_files,
+            }
+
+            get_remote_exec(data)
```

Comment:
Do we want to do something like the following so we can capture results for some comparison or logging?
```suggestion
            remote_emsg, remote_exception = get_remote_exec(data)
```

Note: I know that we said that the main point of this iteration would be the toggle, but I wonder if we might not want some basic info log to start, that possibly just has a boolean for whether the emsg matches, and another for the exception? Thoughts?

Reply:
Yeah, the reason why it doesn't store the data right now is because it doesn't do anything with the results yet and would get complaints from pylint from unused variables.

Reply:
If you want to leave as-is and iterate on this later, that is fine. Once we have observability on the new service, at least we'd have some insight that the toggle is working.